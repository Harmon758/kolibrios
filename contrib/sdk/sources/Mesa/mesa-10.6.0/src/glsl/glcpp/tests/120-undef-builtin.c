#undef __LINE__
#undef __FILE__
#undef __VERSION__
