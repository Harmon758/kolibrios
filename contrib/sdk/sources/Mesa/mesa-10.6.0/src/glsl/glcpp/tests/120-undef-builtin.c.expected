0:1(1): preprocessor error: Built-in (pre-defined) macro names cannot be undefined.
0:2(1): preprocessor error: Built-in (pre-defined) macro names cannot be undefined.
0:3(1): preprocessor error: Built-in (pre-defined) macro names cannot be undefined.



